src/components/Typography/BodyX/BodyX.stories.tsx

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright 2023 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { customCopyable, customEllipsis, loremIpsum } from '../Typography.mocks'
+import { BodySizes } from './BodyX.types'
+import { BodyX } from './BodyX'
+import { Typography } from '..'
+
+const { S } = BodySizes
+
+const bodySLongText = ['BodyS', loremIpsum].join(' | ')
+const bodyMLongText = ['BodyM', loremIpsum].join(' | ')
+const bodyLLongText = ['BodyL', loremIpsum].join(' | ')
+const bodySBoldLongText = ['BodyS Bold', loremIpsum].join(' | ')
+const bodyMBoldLongText = ['BodyM Bold', loremIpsum].join(' | ')
+const bodyLBoldLongText = ['BodyL Bold', loremIpsum].join(' | ')
+
+const bodySEllipsisWithTooltip = { rows: 3, tooltip: bodySLongText }
+const bodyMEllipsisWithTooltip = { rows: 3, tooltip: bodyMLongText }
+const bodyLEllipsisWithTooltip = { rows: 3, tooltip: bodyLLongText }
+const bodySBoldEllipsisWithTooltip = { rows: 3, tooltip: bodySBoldLongText }
+const bodyMBoldEllipsisWithTooltip = { rows: 3, tooltip: bodyMBoldLongText }
+const bodyLBoldEllipsisWithTooltip = { rows: 3, tooltip: bodyLBoldLongText }
+
+const meta = {
+  title: 'Components/Typography/BodyX',
+  component: BodyX,
+  args: {
```

Comment:
What do you think about adding in the design tab the link to fontsource? (google-font is not embeddable)

```
const fontUrl = 'https://fontsource.org/fonts/inter'

...

parameter: {
       design: { type: 'iframe', name: 'Font Definition', url: fontUrl },
}
```